### ISSUE TITLE
Add min, max, and median projection parametrized by a window size to LN_INTPRO

### ISSUE THREAD

**Issue by ofgulban** [layerfMRI/LAYNII#87]
It would be very handy to have minimum, maximum, and median, intensity projection over a selected number of slices and axes to e.g. highlight the veins in T2*w images.

**Comment by layerfMRI** [comment]
Yes that would be nice, handy,and easy to do. I guess the max and min are already in LN_INTPRO. And the mean is in LN_DIRECTIONALSMOOTH (dependent on the selectable smoothing kernel). But we do not have median, yet, I think.

**Comment by ofgulban** [comment]
Ah, I totally forget about `LN_INTPRO` :D 
I will repurpose that one and will not make a new program

**Comment by ofgulban** [comment]
No needed anymore
